Assets/LeapMotion/Scripts/Hands/IHandModel.cs

```diff
@@ -18,7 +18,7 @@ public abstract class IHandModel : MonoBehaviour {
       get { return isTracked; }
     }
 
-    public abstract Chirality Handedness { get; }
+    public abstract Chirality Handedness { get; set; }
```

Comment:
It is not necessary to add the ability to set handedness to the base interface if all you need is the ability to set the handedness of your rigged hands.